contrib/vcl/source/vcl_interface.cc

```diff
@@ -26,7 +21,7 @@ const int MaxNumEpollEvents = 128;
  * Envoy worker epoll session handles by VCL worker index, i.e., `vppcom_worker_index()`. Each
  * worker uses its respective handle to retrieve session events from VCL via `vppcom_epoll_wait()`.
  */
-uint32_t epoll_handles[MaxNumWorkers];
+std::vector<uint32_t> epoll_handles;
```

Comment:
Dynamic containers in static storage [should be avoided](https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables) with `MUTABLE_CONSTRUCT_ON_FIRST_USE`.

Reply:
Should've thought of that. Fixed.